portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/ABTest.testcase

```diff
@@ -0,0 +1,87 @@
+@component-name = "portal-analytics-cloud"
+definition {
+
+	property analytics.cloud.release = "true";
+	property analytics.cloud.upstream = "true";
+	property portal.release = "false";
+	property portal.upstream = "true";
+	property property.group = "analytics.cloud.profile";
+	property testray.main.component.name = "Analytics Cloud";
+
+	setUp {
+		task ("Copy the token from AC") {
+			ACUtils.launchAC();
+
+			ACSettings.takeTokenFromAnalyticsCloud();
+		}
+
+		task ("Set up instance and sign in DXP") {
+			TestCase.setUpPortalInstance();
+
+			ACUtils.launchDXP();
+		}
+
+		task ("Add a new site with a public widget page") {
+			JSONGroup.addGroup(groupName = "Site Name");
+
+			ACUtils.addSiteAndPage(
+				groupName = "Site Name",
+				layoutName = "AC Page",
+				type = "content");
```

Comment:
Hey @joshitagaki, could you explain to me why we need to pass this type of page to the A/B test?

Reply:
When we create a site it is created as a widget page. Widget pages do not have the functionality to create an ab test. Adding the type to content makes the site created into a content page which gives it the ab test function

Reply:
Ohhh nice, thanks for the explanation 😄